packages/sdk/README.md

```diff
@@ -38,6 +39,129 @@ from config import config
+
+---
+
+## Development
+
+### Installing
+
+#### Prerequisites  
+  
+* [Python 3.6.5](https://www.python.org/downloads/release/python-365/)  
+* [Node 8+ w/npm](https://nodejs.org/en/download/)
+
+---
+
+* Clone the git repository  
+```bash  
+$ git clone git@github.com:singnet/snet-cli.git
+$ cd snet-cli/snet_sdk
+```  
+  
+* Install development/test blockchain dependencies  
+```bash  
+$ ./scripts/blockchain install
+```
+  
+* Install the package in development/editable mode  
+```bash  
+$ pip install -e .
+```
+
+### Versioning  
+  
+We use [SemVer](http://semver.org/) for versioning. For the versions available, see the
+[tags on this repository](https://github.com/singnet/snet-cli/tags).   
+  
+## License  
+  
+This project is licensed under the MIT License - see the
+[LICENSE](https://github.com/singnet/snet-cli/blob/master/snet_sdk/LICENSE) file for details.
+# snet-sdk-python
+  
+SingularityNET SDK for Python
+  
+## Getting Started  
+  
+These instructions are for the development and use of the SingularityNET SDK for Python.
+
+### Core concepts
+  
+The SingularityNET SDK allows you to make calls to SingularityNET services programmatically from your application.  
+To communicate between clients and services, SingularityNET uses [gRPC](https://grpc.io/).  
+To handle payment of services, SingularityNET uses [Ethereum state channels](https://dev.singularitynet.io/docs/concepts/multi-party-escrow/).  
+The SingularityNET SDK abstracts and manages state channels with service providers on behalf of the user and handles authentication with the SingularityNET services.
+
+### Usage
+  
+To call a SingularityNET service, the user must be able to deposit funds (AGI tokens) to the [Multi-Party Escrow](https://dev.singularitynet.io/docs/concepts/multi-party-escrow/) Smart Contract.  
+To deposit these tokens or do any other transaction on the Ethereum blockchain, the user must possess an Ethereum identity with available Ether.
+
+
+To interact with SingularityNET services, you must compile the appropriate client libraries for that service.   
+To generate the client libraries to use in your application, you need the SingularityNET Command Line Interface, or CLI, which you can download from PyPi, see [https://github.com/singnet/snet-cli#installing-with-pip](https://github.com/singnet/snet-cli/snet_cli#installing-with-pip)
+  
+Once you have the CLI installed, run the following command:
+```bash
+snet sdk generate-client-library python <org_id> <service_id>
+```
+  
+Optionally, you can specify an output path; otherwise it's going to be `./client_libraries/python/<registry_address>/<org_id>/<service_id>`.  
+You should move or copy these generated files to the root of your project.
+  
+Once you have installed the snet-sdk in your current environment and it's in your PYTHONPATH, you should import it and create an instance of the base sdk class:
+
+```python
+from snet_sdk import SnetSDK
```

Comment:
this is the abandoned one right? the one from [singnet/snet-sdk-python](https://github.com/singnet/snet-sdk-python)?
in this module it is `from snet import sdk` ?